extensions/adobe/experience/aam-experienceevent.example.1.json

```diff
@@ -22,6 +22,10 @@
       }
     ]
   },
+  "xdm:userActivityRegion": {
+    "xdm:dataCenterLocation": "VA6",
+    "xdm:captureTimestamp": "2017-09-26T15:52:20+00:00"
```

Comment:
Why this timestamp will be different from event timestamp?

Reply:
there can be a difference between when action actually happened and when it was captured on edge.

Reply:
A prime example is mobile hits. It is possible to send events to the system in batch (mobile phone comes online again) where the event timestamps might be in the past.